doc/globals.md

```diff
@@ -0,0 +1,573 @@
+| <tt>$&</tt> | <tt>$MATCH</tt>            | Matched substring; set by matcher method.        |
+| <tt>$`</tt> | <tt>$PRE_MATCH</tt>        | Substring left of match; set by matcher method.  |
+| <tt>$'</tt> | <tt>$POST_MATCH</tt>       | Substring right of match; set by matcher method. |
+| <tt>$+</tt> | <tt>$LAST_PAREN_MATCH</tt> | Last group matched; set by matcher method.       |
+| <tt>$1</tt> |                            | First group matched; set by matcher method.      |
+| <tt>$2</tt> |                            | Second group matched; set by matcher method.     |
+| <tt>$</tt>n |                            | nth group matched; set by matcher method.        |
+
+### Separators
+
+| Variable             | English                           | Contains                                         |
+|----------------------|-----------------------------------|--------------------------------------------------|
+| <tt>$/</tt>          | <tt>$INPUT_RECORD_SEPARATOR</tt>  | Input record separator; initially newline.       |
+| <tt>$\\\\\\\\</tt>   | <tt>$OUTPUT_RECORD_SEPARATOR</tt> | Output record separator; initially <tt>nil</tt>. |
+
+### Streams
+
+| Variable         | English                                   | Contains                                                  |
+|------------------|-------------------------------------------|-----------------------------------------------------------|
+| <tt>$stdin</tt>  |                                           | Standard input stream; initially <tt>STDIN</tt>.          |
+| <tt>$stdout</tt> |                                           | Standard input stream; initially <tt>STDIOUT</tt>.        |
+| <tt>$stderr</tt> |                                           | Standard input stream; initially <tt>STDERR</tt>.         |
+| <tt>$<</tt>      | <tt>$DEFAULT_INPUT</tt>                   | Default standard input; <tt>ARGF</tt> or <tt>$stdin</tt>. |
+| <tt>$></tt>      | <tt>$DEFAULT_OUTPUT</tt>                  | Default standard output; initially <tt>$stdout</tt>.      |
+| <tt>$.</tt>      | <tt>$INPUT_LINE_NUMBER</tt>, <tt>$NR</tt> | Input position of most recently read stream.              |
+| <tt>$_</tt>      | <tt>$LAST_READ_LINE</tt>                  | String from most recently read stream.                    |
+
+### Processes
+
+| Variable                                       | English                             | Contains                                               |
+|------------------------------------------------|-------------------------------------|--------------------------------------------------------|
+| <tt>$0</tt>                                    |                                     | Initially, the name of the executing program.          |
+| <tt>$*</tt>                                    | <tt>$ARGV</tt>                      | <tt>ARGV</tt>.                                         |
```

Comment:
I think the description for ARGV is a bit too basic and hard to understand. Maybe something like `Points to the <tt>ARGV</tt> array.`?

Reply:
Done.